spec/lib/subjects/remover_spec.rb

```diff
@@ -65,6 +74,35 @@
         expect { Subject.find(subject.id) }.to raise_error(ActiveRecord::RecordNotFound)
       end
 
+      context "with a non-zero count sws record" do
+        let(:linked_sws) do
+          create(
+            :subject_workflow_status,
+            workflow: workflow,
+            subject: subject,
+            classifications_count: 10
+          )
+        end
+        it "should not remove a subject that has a non-zero count a sws record" do
+          expect(remover.cleanup).to be_falsey
+        end
+      end
+
+      context "with a non-zero count sws record" do
```

Comment:
If these are the same context, maybe use describes to differentiate the two tests, repeating the context seems awkward.